src/Sulu/Bundle/AdminBundle/Resources/views/CsvExport/form.html.twig

```diff
@@ -75,4 +75,7 @@
             </div>
         </div>
     </div>
+
+    {% block additions %}
```

Comment:
I would prefer to also add blocks around the fields e.g. `{% block delimiter %}` so he can edition this fields to.

We could also make just a block `content` around all fields when he overwrite it he can use the parent function 

e.g.:

```twig
{% block content %}
    {# he can add fields before #}
    {{ parent() }}
    {# he can add fields after #}
{% endblock %}
```